doc/api/stream.md

```diff
@@ -2095,15 +2105,16 @@ added:
   - v17.4.0
```

Comment:
All the iterator helpers are experimental so we can break them without a deprecation cycle if the spec changes.

Unfortunately that spec is still Stage 2 (since it was split from the stage 4 sync one): https://github.com/tc39/proposal-async-iterator-helpers 

This breaks that